src/hotspot/share/classfile/classLoaderDataGraph.hpp

```diff
@@ -97,11 +97,6 @@ class ClassLoaderDataGraph : public AllStatic {
   // Called from VMOperation
   static void walk_metadata_and_clean_metaspaces();
 
-  // VM_CounterDecay iteration support
```

Comment:
Oh, nice, I think this was removed with the sweeper.

There's also a function InvocationCounter::decay() that can also be removed (maybe more but that's what I found for now).